jenkins-pipelines/perf-regression-latency-125gb-with-ebs-volume.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression-latency-125gb.yaml","configurations/ebs/ebs-gp2-2v-4tb.yaml"]'''
```

Comment:
@aleksbykov where is this file: "perf-regression-latency-125gb.yaml" ?
There is also a mess with the throughput files.
I suggest to remove all throughput tests from this PR, we will merge it and then send a new PR only with throughput tests.

Reply:
@roydahan , the `perf-regression-latency-125gb.yaml` is here: https://github.com/scylladb/scylla-cluster-tests/pull/2984/files#diff-85d20f259ad68c2d68e5aafed95ab2d9288554a15fd6af989077c02edabb0f30